dll/win32/netapi32/wksta_new.c

```diff
@@ -327,6 +327,7 @@ NetGetJoinInformation(
     if (lpNameBuffer == NULL || BufferType == NULL)
         return ERROR_INVALID_PARAMETER;
 
+    *lpNameBuffer = NULL;
```

Comment:
Maybe add a HACK comment, as `NetpGetJoinInformation()` already has `*NameBuffer = NULL;`.
https://git.reactos.org/?p=reactos.git;a=blob;f=base/services/wkssvc/domain.c;hb=e8277e51e2ec9c13721b68cd8062b3b7d933d255#l69

Reply:
Should the fix be to remove `in` from https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/idl/wkssvc.idl;hb=e8277e51e2ec9c13721b68cd8062b3b7d933d255#l536 ? Or is this somehow unsupported by widl/rpcrt4?

Reply:
The annotation matches what Windows has: https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-wkst/12d7af34-9efc-4f39-ab88-a8d8e7bcbdc2 and https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-wkst/9fdbc753-0397-4236-bbfc-a380f9d23789